examples/text-to-speech/elevenlabs_tts.py

```diff
@@ -1,37 +1,105 @@
 import asyncio
 import logging
+from typing import Optional
 
 from livekit import rtc
 from livekit.agents import (
     JobContext,
     JobRequest,
     WorkerOptions,
     cli,
+    tts,
 )
 from livekit.plugins import elevenlabs
 
 
+def _text_to_chunks(text: str) -> list[str]:
+    """Split the text into chunks of 2, 3, and 4 words"""
+    sizes = [2, 3, 4]
+    chunks, i = [], 0
+
+    for size in sizes:
+        while i + size <= len(text):
+            chunks.append(text[i : i + size])
+            i += size
+
+    chunks.append(text[i:])  # remaining
+    return chunks
+
+
+async def _playout_task(
+    playout_q: asyncio.Queue, audio_source: rtc.AudioSource
+) -> None:
+    """Playout audio frames from the queue to the audio source"""
+    while True:
+        frame = await playout_q.get()
+        if frame is None:
+            break
+
+        await audio_source.capture_frame(frame)
+
+
 async def entrypoint(job: JobContext):
-    tts = elevenlabs.TTS(
+    # use another voice for this demo
+    # you can get a list of the voices using 'await tts_11labs.list_voices()'
+    voice = elevenlabs.Voice(
+        id="ODq5zmih8GrVes37Dizd",
+        name="Patrick",
+        category="premade",
+    )
+
+    tts_11labs = elevenlabs.TTS(
         model_id="eleven_multilingual_v2",
+        voice=voice,
     )
 
-    source = rtc.AudioSource(tts.sample_rate, tts.num_channels)
+    source = rtc.AudioSource(tts_11labs.sample_rate, tts_11labs.num_channels)
     track = rtc.LocalAudioTrack.create_audio_track("agent-mic", source)
     options = rtc.TrackPublishOptions()
     options.source = rtc.TrackSource.SOURCE_MICROPHONE
     await job.room.local_participant.publish_track(track, options)
 
     await asyncio.sleep(1)
     logging.info('Saying "Bonjour, comment allez-vous?"')
-    async for output in tts.synthesize("Bonjour, comment allez-vous?"):
```

Comment:
we gonna do french by default? hah

Reply:
we have another example that isn't french, it is just to show that we can do whatever language here, but yeab I'll mb separate it in another file